data/events/pitercss_conf-2017

```diff
@@ -0,0 +1,28 @@
+{
+    "accessibility": "",
+    "cfp_end": "",
+    "code_of_conduct": "https://pitercss.com/code-of-conduct/",
+    "comment": "",
+    "diversitytickets": "",
+    "diversitytickets_text": "",
+    "diversitytickets_url": "",
+    "event_end": "2017-06-16",
+    "event_start": "2017-06-16",
+    "facebook": "https://www.facebook.com/pitercssconf/",
+    "hashtag": "pitercss_conf",
+    "languages": [
+        "English",
+        "Russian"   
+    ],
+    "location": {
+        "city": "Saint Petersburg",
+        "country": "Russia",
+        "state": ""
+    },
+    "name": "PiterCSS Conference-2017",
```

Comment:
No need to have the year in the name.

Reply:
May be worth changing the docs/EVENTS.md file as it says:

"The name might need to include the country/city and the year. Check similar events.  It should end with the year of the event. (e.g. 2016)"

Reply:
Sure, you can send a PR to fix the EVENTS.md